src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -568,6 +568,7 @@ public function renderRelationElement($element, FieldDescriptionInterface $field
     public function getUrlSafeIdentifier($model, ?AdminInterface $admin = null)
     {
         if (null === $admin) {
+            /** @phpstan-var class-string $class */
             $class = ClassUtils::getClass($model);
```

Comment:
Couldn't we fix this by defining class-string on the getClass method? Or is it not part of sonata? I am currently on a phone 📱 and can't double check easily, that's why I ask

Reply:
It's `Doctrine` class utils

Reply:
Do you want to make a PR: https://github.com/doctrine/common/blob/7198b184394431e443baf0e00012d96b920c3c74/lib/Doctrine/Common/Util/ClassUtils.php ?

Reply:
I'll do it: https://github.com/doctrine/common/pull/920